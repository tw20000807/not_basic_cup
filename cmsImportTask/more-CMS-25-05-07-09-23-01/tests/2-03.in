xBKhCirDv7dWDvJlxKTtuZimwF4Zi9x1
1
7 3
1832142813 441793603 283171512 1969528057 1325074106 925349653 865420769
385496033 706888392 51206914 1832142813 1969528057 283171512 925349653 865420769 1325074106 441793603
