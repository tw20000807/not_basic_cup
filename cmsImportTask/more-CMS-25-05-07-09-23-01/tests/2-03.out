du3W59OSqKqC9FuBgxi0VkGO13qNOHq6
51206914 385496033 385496033
